examples/scripts/deconv_modl_train_foam.py

```diff
@@ -0,0 +1,348 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+# This file is part of the SCICO package. Details of the copyright
+# and user license can be found in the 'LICENSE.txt' file distributed
+# with the package.
+
+r"""
+Training of MoDL for Deconvolution
+==================================
```

Comment:
Same comment as previous example: filename implies training only, etc.

Reply:
Keeping 'train' in the filename for compatibility with the CI workflow: train scripts should be skipped, the filename is used to identify such scripts. I also used 'foam1' instead of 'foam'.